src/component/maincomponent.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import Header from './header';
+import Content from './content';
+import Footer from './footer'
+
+
+class Main extends Component{
+    constructor(props){
+        super(props);
+        this.state={
+            FirstName: ['pedro','devon'],
+            LastName: ['sanchez','gonzalez']
```

Comment:
Convention in javascript is to use camelCase so should be like this: 
```suggestion
            firstName: ['pedro','devon'],
            lastName: ['sanchez','gonzalez']
```